fastlane/lanes/build.rb

```diff
@@ -0,0 +1,427 @@
+      external: true
+    )
+
+    ios_build_preflight
+
+    build_and_upload_itc(
+      skip_prechecks: true,
+      skip_confirm: options[:skip_confirm],
+      beta_release: options[:beta_release],
+      create_release: options[:create_gh_release]
+    )
+  end
+
+  # Builds the Jetpack app and uploads it to TestFlight, for beta-testing or final release
+  #
+  # @called_by CI
+  #
+  desc 'Builds and uploads Jetpack to TestFlight for distribution'
+  lane :build_and_upload_jetpack_for_app_store do
+    jetpack_appstore_code_signing
+
+    gym(
+      scheme: 'Jetpack',
+      workspace: WORKSPACE_PATH,
+      clean: true,
+      export_team_id: get_required_env('EXT_EXPORT_TEAM_ID'),
+      output_directory: BUILD_PRODUCTS_PATH,
+      derived_data_path: DERIVED_DATA_PATH,
+      export_options: { method: 'app-store' }
+    )
+
+    testflight(
+      skip_waiting_for_build_processing: true,
+      team_id: '299112',
+      api_key_path: APP_STORE_CONNECT_KEY_PATH
+    )
+
+    sentry_upload_dsym(
+      auth_token: get_required_env('SENTRY_AUTH_TOKEN'),
+      org_slug: 'a8c',
+      project_slug: 'jetpack-ios',
+      dsym_path: lane_context[SharedValues::DSYM_OUTPUT_PATH]
+    )
+  end
+
+  # Builds the app and uploads it to App Center
+  #
+  # @option [Boolean] skip_confirm (default: false) If true, avoids any interactive prompt
+  #
+  # @called_by CI
+  # @calls build_and_upload_internal
+  #
+  desc 'Builds and uploads for distribution to App Center'
+  lane :build_and_upload_app_center do |options|
+    ios_build_prechecks(
+      skip_confirm: options[:skip_confirm],
+      internal: true,
+      external: true
+    )
+
+    ios_build_preflight
+
+    build_and_upload_internal(
+      skip_prechecks: true,
+      skip_confirm: options[:skip_confirm]
+    )
+  end
+
+  # Builds the WordPress app for an Installable Build ("WordPress Alpha" scheme), and uploads it to AppCenter
+  #
+  # @called_by CI
+  #
+  desc 'Builds and uploads an installable build'
+  lane :build_and_upload_installable_build do
+    sentry_check_cli_installed
+
+    alpha_code_signing
+
+    # Get the current build version, and update it if needed
+    version_config_path = File.join(PROJECT_ROOT_FOLDER, 'config', 'Version.internal.xcconfig')
+    versions = Xcodeproj::Config.new(File.new(version_config_path)).to_hash
+    build_number = generate_installable_build_number
+    UI.message("Updating build version to #{build_number}")
+    versions['VERSION_LONG'] = build_number
+    new_config = Xcodeproj::Config.new(versions)
+    new_config.save_as(Pathname.new(version_config_path))
+
+    gym(
+      scheme: 'WordPress Alpha',
+      workspace: WORKSPACE_PATH,
+      export_method: 'enterprise',
+      clean: true,
+      output_directory: BUILD_PRODUCTS_PATH,
+      output_name: 'WordPress Alpha',
+      derived_data_path: DERIVED_DATA_PATH,
+      export_team_id: ENV['INT_EXPORT_TEAM_ID'],
+      export_options: { method: 'enterprise' }
```

Comment:
🤔 the `export_method` and `export_options` values seemed redundant but [this test](https://github.com/fastlane/fastlane/blob/e874a47c6e2e0e61590a03d3b71e75e5a505d1ce/gym/spec/error_handler_spec.rb#L58-L86) in Fastlane suggests ti might actually need or use both. 

---

How do you feel about groping the values for slightly improved local reasoning?

```suggestion
      clean: true,
      output_directory: BUILD_PRODUCTS_PATH,
      output_name: 'WordPress Alpha',
      derived_data_path: DERIVED_DATA_PATH,
      export_team_id: ENV['INT_EXPORT_TEAM_ID'],
      export_method: 'enterprise',
      export_options: { method: 'enterprise' }
```